scipy/fft/_helper.py

```diff
@@ -37,22 +37,22 @@ def next_fast_len(target, real=False):
 
     Examples
     --------
-    On a particular machine, an FFT of prime length takes 17 ms:
+    On a particular machine, an FFT of prime length takes 11.4 ms:
 
     >>> from scipy import fft
     >>> min_len = 93059  # prime length is worst case for speed
     >>> a = np.random.randn(min_len)
     >>> b = fft.fft(a)
 
     Zero-padding to the next regular length reduces computation time to
-    1.3 ms, a speedup of 13 times:
+    2.1 ms, a speedup of 5.4 times:
 
     >>> fft.next_fast_len(min_len)
```

Comment:
The input is real, so this should be passing `real=True`:
```suggestion
    >>> fft.next_fast_len(min_len, True)
```

This smaller number has prime factors 7 and 11 which aren't specialised for real inputs.

I also just noticed that keyword arguments don't work with `next_fast_len` though. Will fix in another PR.